### PR TITLE
After getting grpc-web working after protoc upgrade

### PR DESCRIPTION
**Pull request by SeanCondon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +12,7 @@ For the moment (Jul '19) the Configuration GUI is the main target, but generally
 the architecture will support UI extensions for any project, as libraries.
 
 ## Architecture
-The GUI is architected as a Cloud Native application to be run on Kubernetes
-accessing the back end modules through gRPC (and in the case of the Config GUI
-additionally through gNMI)
+See [architecture.md](./docs/architecture.md) for an architectural discussion.
 
-The front end technology is [Angular](https://angular.io) and [gRPC Web](https://github.com/grpc/grpc-web)
```
Comment (ray-milkey) [inline README.md]: Extra 'is' here

File: docs/architecture.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# onos-gui Architecture
+The GUI is architected as a Cloud Native application to be run on Kubernetes
+accessing the back end modules through gRPC (and in the case of the Config GUI
+additionally through gNMI)
+
+The front end technology is [Angular](https://angular.io) and [gRPC Web](https://github.com/grpc/grpc-web)
```
Comment (ray-milkey) [inline docs/architecture.md]: Extra 'is' here

File: docs/architecture.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+is used to access the back end systems. [Nginx](https://nginx.org/) acts as a
+Proxy, Envoy acts as a grpc-web proxy and all are deployed on [Docker](https://www.docker.com/community/open-source).
```
Comment (ray-milkey) [inline docs/architecture.md]: Is there a linkable page for Envoy we can put here?

Comment (SeanCondon) [reply docs/architecture.md]: done

File: docs/architecture.md (hunk 2 above)
Comment (SeanCondon) [reply docs/architecture.md]: done

**Pull request merged**